src/scripts/checkMarkdownLanguage.mjs

```diff
@@ -13,9 +13,9 @@ const getMarkdownFilesWithLanguage = () => {
     const markdownFilesWithLanguageCommand = spawn("git", [
       ...commonDiffCommand,
       "-G",
-      "lang:",
+      "lang:[ ]*\\w+",
```

Comment:
Every markdown file under `src/content` needs to have a non-empty lang attribute